### PR TITLE
add modal in section about

### PR DESCRIPTION
**Pull request by Alisa-Bogatyryova**

### CODE REVIEW

File: src/sass/layouts/_about.scss [hunk 1]
```diff
@@ -186,3 +186,44 @@
...
+    max-width: 50%;
+  }
+  @include for-desktop-up {
+    margin: 40px;
+    font-size: 16px;
+  }
+}
+.modal-about__title {
+  margin-bottom: 10px;
+  margin-top: 10px;
```
Comment (izachok) [inline src/sass/layouts/_about.scss]: is it possible to use only margin-bottom?

**Review by izachok — CHANGES_REQUESTED** [review]
Please restore index.thml

File: src/sass/layouts/_about.scss [hunk 2]
```diff
@@ -186,3 +186,44 @@
...
+  display: none;
+  @include for-tablet-up {
+    display: block;
+    width: 143px;
+  }
+  @include for-desktop-up {
+    width: 236px;
+  }
+}
+.modal-about__position {
```
Comment (vixxxyg) [inline src/sass/layouts/_about.scss:200]: Можно заменить на миксин @mixin flex-centered

File: src/sass/layouts/_about.scss (hunk 1 above)
Comment (Alisa-Bogatyryova) [reply src/sass/layouts/_about.scss]: исправлено

**Review by izachok — APPROVED** [review]

**Pull request merged**